src/core/mount.c

```diff
@@ -856,7 +856,7 @@ static void mount_dump(Unit *u, FILE *f, const char *prefix) {
 
 static int mount_spawn(Mount *m, ExecCommand *c, PidRef *ret_pid) {
         _cleanup_(exec_params_shallow_clear) ExecParameters exec_params = EXEC_PARAMETERS_INIT(
-                        EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_APPLY_TTY_STDIN);
+                        EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_APPLY_TTY_STDIN|EXEC_SETUP_CREDENTIALS_FRESH);
```

Comment:
Hmm, it seems unnecessary to use _FRESH variant? We only spawn one process for mounting.

But please don't set this for unmount processes, to stay in line with service logic.